pkg/config/security/security.go

```diff
@@ -181,3 +183,35 @@ func validateMapKey(key string) error {
 	}
 	return fmt.Errorf("bad key (%s): should have format a[b]", key)
 }
+
+// ValidCipherSuites contains a list of all ciphers supported in Gateway.server.tls.cipherSuites
+// Extracted from: `bssl ciphers -openssl-name ALL | rg -v PSK`
+var ValidCipherSuites = sets.NewSet(
+	"ECDHE-ECDSA-AES128-GCM-SHA256",
+	"ECDHE-RSA-AES128-GCM-SHA256",
+	"ECDHE-ECDSA-AES256-GCM-SHA384",
+	"ECDHE-RSA-AES256-GCM-SHA384",
+	"ECDHE-ECDSA-CHACHA20-POLY1305",
+	"ECDHE-RSA-CHACHA20-POLY1305",
+	"ECDHE-ECDSA-AES128-SHA",
+	"ECDHE-RSA-AES128-SHA",
+	"ECDHE-ECDSA-AES256-SHA",
+	"ECDHE-RSA-AES256-SHA",
+	"AES128-GCM-SHA256",
+	"AES256-GCM-SHA384",
+	"AES128-SHA",
+	"AES256-SHA",
+	"DES-CBC3-SHA",
```

Comment:
Hi @howardjohn, we were trying to enable some of the cipher suites in our Istio config but we face 2 issues. 
1. Even after specifying these values in our gateway.yaml file, we do not see some of them when we run the scanner on our endpoint here for TLS 1.2: https://www.ssllabs.com/ssltest/
2. Is there a way to specify the cipher suites for TLS 1.3? We do see some of the cipher suites show up on the above scanner even after not specifying them in our config. Some of them are not allowed as per our company policy. Is there a way to fix this?

Reply:
1. Hard to say without seeing the configdump, logs, etc
2. boringssl doesn't allow custom cipher suites in TLS 1.3. So if you want this in Istio you would need to change boringssl.